projects/storefrontlib/src/recipes/config/b2b-occ-config.ts

```diff
@@ -8,9 +8,9 @@ export const defaultB2bOccConfig: OccConfig = {
         addEntries: 'orgUsers/${userId}/carts/${cartId}/entries',
         setDeliveryAddress:
           'orgUsers/${userId}/carts/${cartId}/addresses/delivery',
-        // TODO(#8877): Is this a hack for an API? Shouldn't user pass this data?
-        placeOrder: 'orgUsers/${userId}/orders?termsChecked=true',
-        scheduleReplenishmentOrder: 'orgUsers/${userId}/replenishmentOrders',
+        placeOrder: 'orgUsers/${userId}/orders?fields=FULL',
```

Comment:
Do we always need full?

Reply:
it was already there before just moved it out of the adapter